Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -291,6 +291,11 @@ def audit_file
 
     problem "File should end with a newline" unless text.trailing_newline?
 
+    if !Dir.glob("#{formula.path.to_s.gsub(/.rb$/, "")}@*.rb").empty? &&
```

Comment:
I think your usual preference for multiple returns might make these conditions easier to understand.